src/scene_graph/camera_node.cpp

```diff
@@ -55,6 +55,16 @@ namespace wr
 		return std::pair<float, float>(m_projection_offset_x, m_projection_offset_y);
 	}
 
+	void CameraNode::SetOrthographicResolution(std::uint32_t width, std::uint32_t height)
+	{
+		m_ortho_res.m_width = static_cast<int>(width);
```

Comment:
The reason for this is because the window counts negative on screen, without this the Y coordinate will be flipped.